Source/OxyPlot/Series/AreaSeries.cs

```diff
@@ -238,11 +243,27 @@ public override void Render(IRenderContext rc)
             var clippingRect = this.GetClippingRect();
             rc.SetClip(clippingRect);
 
-            var chunksOfPoints = new List<List<ScreenPoint>>();
-            var chunksOfPoints2 = new List<List<ScreenPoint>>();
+            var areaContext = new AreaRenderContext
```

Comment:
this is a new pattern, but I think this is something that can be used in more places. Maybe more of the rendering code could use composition over inheritance?